SurveyApp/Extensions/UIKit/UIImage/UIImage+URL.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIImage+URL.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 6/21/21.
+//
+
+import UIKit
+
+extension UIImageView {
+
+    func loadUrl(url: URL) {
```

Comment:
I recommend you use [AlamofireImage](https://github.com/Alamofire/AlamofireImage) to load image from remote URL, it will save you a lot of time 😁